common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -90,6 +104,24 @@ class AccountInfo extends React.Component<Props, State> {
           </div>
           <div className="AccountInfo-address-wrapper">
             <div className="AccountInfo-address-addr">{address}</div>
+            <CopyToClipboard onCopy={this.onCopy} text={address}>
+              <div className="copy-clipboard" title="copy to clipboard">
+                <svg
+                  aria-hidden="true"
+                  data-prefix="far"
+                  data-icon="copy"
+                  role="img"
+                  viewBox="0 0 448 512"
+                  className="svg-inline--fa fa-copy fa-w-14 fa-2x"
+                >
+                  <path
+                    fill={this.state.currentColor}
+                    d="M433.941 65.941l-51.882-51.882A48 48 0 0 0 348.118 0H176c-26.51 0-48 21.49-48 48v48H48c-26.51 0-48 21.49-48 48v320c0 26.51 21.49 48 48 48h224c26.51 0 48-21.49 48-48v-48h80c26.51 0 48-21.49 48-48V99.882a48 48 0 0 0-14.059-33.941zM266 464H54a6 6 0 0 1-6-6V150a6 6 0 0 1 6-6h74v224c0 26.51 21.49 48 48 48h96v42a6 6 0 0 1-6 6zm128-96H182a6 6 0 0 1-6-6V54a6 6 0 0 1 6-6h106v88c0 13.255 10.745 24 24 24h88v202a6 6 0 0 1-6 6zm6-256h-64V48h9.632c1.591 0 3.117.632 4.243 1.757l48.368 48.368a6 6 0 0 1 1.757 4.243V112z"
+                  />
+                </svg>
```

Comment:
You can use `<i className="fa fa-copy"/>` instead, we already use font awesome elsewhere this way.